.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+on: [ push ]
+
+jobs:
+  migration_job:
+    runs-on: ubuntu-latest
+    name: CI migration
+    steps:
+      # Checkout the source code
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Run the migration with Postgres.ai Action
+      - name: DB migration with DLE
+        uses: agneum/migration-action@v0.0.10
+        id: db-migration
+        with:
+          dbname: test
+          commands: |
+            # This is only for testing purposes since
+            # the demo database already has tables
+            rails db:reset_schema
+            echo 'Reset the database successfully.'
+
+            rails db:migrate
+            echo 'Forward migration completed successfully.'
+
+            rails db:rollback
```

Comment:
for the "real thing" -- I think it's be nice for users to see two actions -- one for roll forward, one for rollback -- so you can see it in the PR.  Do you know if it's possible to post two notifications in the PR, for a single action run?  (I know we could always have two action runs)